app/src/main/java/com/nextcloud/talk/models/json/chat/ChatUtils.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Nextcloud Talk application
+ *
+ * @author Mario Danic
+ * Copyright (C) 2017-2018 Mario Danic <mario@lovelyhq.com>
+ * @author Marcel Hibbe
+ * Copyright (C) 2021 Marcel Hibbe <dev@mhibbe.de>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.nextcloud.talk.models.json.chat
+
+import java.util.*
+
+class ChatUtils {
+    companion object {
+        fun getParsedMessage(message: String?, messageParameters: HashMap<String?, HashMap<String?, String?>>?):
+                String? {
+            var message = message
```

Comment:
That's uncommon, or?
If you want to you can change parameter to "var message" directly.

Reply:
for function parameters "var" is not allowed in kotlin (https://discuss.kotlinlang.org/t/method-parameters-and-mutable-variables/333). I guess you were thinking about primary constructors for classes where it is allowed.

`var message = message`  is called "shadowing" (https://stackoverflow.com/questions/49680040/why-kotlin-allows-to-declare-variable-with-the-same-name-as-parameter-inside-the), but maybe it's not best practice here.
so i will go for
`var resultMessage = message`
Not sure if this is best practice so i'm open to other ideas..